lib/src/paged_datatable.dart

```diff
@@ -94,9 +98,41 @@ final class PagedDataTable<K extends Comparable<K>, T> extends StatefulWidget {
     this.footer,
     this.filterBarChild,
     this.filters = const <TableFilter>[],
+    this.onRowPressed,
     super.key,
   });
 
+  static _PagedDataTableState<K, T> _of<K extends Comparable<K>, T>(
```

Comment:
I like to use it to reduce the amount of widgets that have to pass down variables through the widget tree. In this case going with passing it down would be fine I guess. As it is only part of the onRowPressed event part of this PR, it will be removed anyway from the new PR